components/layout/Layout.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable unused-imports/no-unused-vars */
```

Comment:
Don't disable these rules for this file. Instead resolve the errors made by eslint.